web-components/src/components/cards/ReviewCard/ReviewCard.tsx

```diff
@@ -33,7 +33,7 @@ export const ReviewCard: React.FC<ReviewCardProps> = ({
         <Label size="sm">{title}</Label>
         <EditButton onClick={onEditClick} />
       </Flex>
-      {children}
+      <FlexCol sx={{ mt: [4, 7], pb: 4, gap: [8, 10] }}>{children}</FlexCol>
```

Comment:
@haveanicedavid Suggesting we add this here instead of each use case. All of these review sections look the same and this makes them consistent.

Reply:
@mhagel ya i’m fine with that. Can always pass an SX override in the future if we need